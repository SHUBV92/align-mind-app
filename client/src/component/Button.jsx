import React from "react"

const Button = (props) => { 

    const { onClick, name } = props

    return (
        <div>
            <button onClick={onClick}>{name}</button>
        </div>
    )
}

export default Button; 