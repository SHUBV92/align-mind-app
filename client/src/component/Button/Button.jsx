import React from "react";

const Button = (props) => {
  const { onClick, name, className } = props;

  return (
    <div>
      <button className={className} onClick={onClick}>
        {name}
      </button>
    </div>
  );
};

export default Button;
