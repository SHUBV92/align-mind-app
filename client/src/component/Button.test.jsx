import React from "react";
import { render } from "@testing-library/react";
import Button from "./Button";

it("should take snapshot", () => {
  const { asFragment } = render(<Button />);

  expect(asFragment(<Button />)).toMatchSnapshot();
});
