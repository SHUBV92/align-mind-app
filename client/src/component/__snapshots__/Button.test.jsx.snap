// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take snapshot 1`] = `
<DocumentFragment>
  <div>
    <button />
  </div>
</DocumentFragment>
`;
